Rakefile

```diff
@@ -4,11 +4,16 @@ Bundler::GemHelper.install_tasks
 begin
   require 'rspec/core/rake_task'
 
-  task :default => :spec
+  task :default => [:unit_spec, :functional_spec]
```

Comment:
@chefsalim not sure i get the question, but it just separates functional and unit tests into separate tasks but the default task (what ci uses) will still run both.